03-bundling/07-vite-lib/01-entrypoints/README.md

```diff
@@ -406,16 +406,6 @@ _./playgrounds/bundler/index.html_
 </html>
 ```
 
-Let's add a `index.js` file:
-
-_./playgrounds/bundler/index.js_
-
-```javascript
-import { myFn } from "my-lib";
-
-myFn();
-```
-
```

Comment:
We don't use this one so I've removed it. It makes not sense because using node here belongs to ESModule playground